pkg/crc/cluster/cluster.go

```diff
@@ -40,10 +40,8 @@ func CheckCertsValidity(driver drivers.Driver) error {
 }
```

Comment:
I would add a short description about the "why" to the commit log (depending on the hypervisor being used, the device name may not be /dev/vda3).